### PR TITLE
Add endpoint to force-upload multiple PIDs to YV with one request

### PR DESCRIPTION
**Pull request by emilssolmanis**
Adds option to POST a JSON list of strings and then does all the concurrent stuff internally. Previously we'd have to script this ourselves via the single-PID endpoint, and since we've had to do that a number of times now, this seems easier.

### CODE REVIEW

File: src/main/java/org/atlasapi/feeds/youview/www/YouViewUploadController.java [hunk 1]
```diff
@@ -137,7 +162,51 @@ public void uploadSchedule(HttpServletResponse response,
...
+                        return Try.exception(e);
+                    }
+                }
+            });
+
+            responses.add(task);
+        }
+
+        ListenableFuture<List<Try>> allTasks = Futures.allAsList(responses);
+        List<Try> allResponses = allTasks.get(10, TimeUnit.MINUTES);
```
Comment (Spartakovic) [inline src/main/java/org/atlasapi/feeds/youview/www/YouViewUploadController.java]: move to constant maybe?

**Comment by Spartakovic** [comment]
lgtm

File: src/main/java/org/atlasapi/feeds/youview/www/YouViewUploadController.java [hunk 2]
```diff
@@ -496,4 +565,40 @@ private void sendOkResponse(HttpServletResponse response, String message) throws
         response.setStatus(HttpServletResponse.SC_OK);
         response.getOutputStream().write(message.getBytes(Charsets.UTF_8));
     }
+
+    private static class Try {
+
+        private final Optional<Exception> exception;
+        private final String data;
+
+        public static Try exception(Exception e) {
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/feeds/youview/www/YouViewUploadController.java]: Code style. Constructors first

**Comment by Dalamar42** [comment]
one minor code style thing, OLGTM

**Pull request merged**